wire-ios-sync-engine/Source/Synchronization/StrategyDirectory.swift

```diff
@@ -67,6 +69,7 @@ public class StrategyDirectory: NSObject, StrategyDirectoryProtocol {
 
         self.requestStrategies = strategies.compactMap({ $0 as? RequestStrategy})
         self.eventConsumers = strategies.compactMap({ $0 as? ZMEventConsumer })
+        self.eventAsyncConsumers = strategies.compactMap({ $0 as? ZMEventAsyncConsumer })
```

Comment:
what do we do with these? they are still used as strategy?

Reply:
Many strategies are both a request generator and an event consumer